pkg/runtime/ecosystem/java.go

```diff
@@ -54,3 +65,54 @@ func (e *Java) Remove(artifact *buildplan.Artifact) error {
 func (e *Java) Apply() error {
 	return nil
 }
+
+func (e *Java) injectClasspath(runtimeJson string) error {
+	bytes, err := fileutils.ReadFile(runtimeJson)
+	if err != nil {
+		return errs.Wrap(err, "Unable to read runtime.json")
+	}
+
+	m := make(map[string]interface{})
+	err = json.Unmarshal(bytes, &m)
+	if err != nil {
+		return errs.Wrap(err, "Unable to unmarshal runtime.json")
+	}
+
+	classpathEnv := map[string]interface{}{
+		"env_name":  "CLASSPATH",
+		"values":    []string{"${INSTALLDIR}/lib"},
+		"join":      "prepend",
+		"inherit":   true,
+		"separator": ":",
+	}
```

Comment:
We should be able to use 

https://github.com/ActiveState/cli/blob/169471b44a507f115252ba0a1eceb100a163e94d/pkg/runtime/internal/envdef/environment.go#L47-L47